apps/condo/migrations/20210720154738-0028_organizationlinkemployeeaccess_organizationlinkemployeeaccesshistoryrecord.js

```diff
@@ -0,0 +1,46 @@
+// auto generated by kmigrator
```

Comment:
The same thing.
If you will tweak number of migration `…organizationlink_organizationlink_historyrecord`, then you should also tweak number of this migration.